explorer/src/api/graphql/config_param.rs

```diff
@@ -528,7 +511,7 @@ impl From<&ConfigParamLib> for ConfigParam {
                 Self::TransactionMaxExpiryEpochs(v.into())
             }
             #[cfg(feature = "evm")]
-            ConfigParamLib::EvmParams(v) => Self::EvmParams(v.into()),
+            ConfigParamLib::EvmParams(_) => unimplemented!("evm support"),
```

Comment:
Are we just going to drop EVM support?

Reply:
well, not drop it, since but we don't really have support for this in jormungandr either: https://github.com/input-output-hk/jormungandr/blob/97f3439d601f99bb65007dcf48013a5905efdf6e/jormungandr/src/explorer/graphql/config_param.rs#L447

But I will just put the same thing here for consistency and to avoid the panic.